### ISSUE TITLE
Search option for xml view

### ISSUE THREAD

**Issue by LeoTT** [process-engine/bpmn-studio#289]
#### Describe your issue

It would be nice to be able to search the xml view for text, because xml is hard on the eyes.
Right now it is impossible to use the browsers search text feature(not sure about windows & linux, but at least for osx's `cmd-f`. #291 ).
But also for the electron app it would be a plus because afaik there is no such feature included by default.

#### How can others reproduce the issue?

Go to xml view and try to search anything.

#### Possible solution

The left toolbar could be replaced with a toolbar just for the xml view (including a search feature).
Right now it shows tools which are only useful for the diagram (and therefore not useful at all at that moment).

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [ ] I've added logs and/or screenshots (if applicable)
- [ ] I've mentioned PRs and issues that relate to this one

**Comment by LeoTT** [comment]
I feel like any good that comes from the xml-view depends on the ability to search it and to copy from it (both of which does not work now). 

When I made this issue a few months ago I could have used that feature in order to debug some bpmn-studio issue. 
What is the xml-view really for? 
There are other applications for it (like searching for text-snippets in the actual file, when the modeler search does not suffice), but they might not seem intuitive and could just as well be handled on the actual file in another editor.

I'm fine with closing this. When I opened the issue back then, it seemed like a real handicap for the xml-view, but I can see why the xml-view is no priority.
